backend/medtagger/api/tasks/business.py

```diff
@@ -1,4 +1,5 @@
 """Module responsible for business logic in all Tasks endpoints."""
+# pylint: disable-msg=too-many-arguments
```

Comment:
Please do it on the function level if needed. Disabling this check here will have an impact on whole module.